doc/Learn/HowTo-UseThemeService.md

```diff
@@ -0,0 +1,56 @@
+---
+uid: Uno.Extensions.ThemeService
+---
+
+# How to use Theme Service
+
+The Uno Extensions library includes a `ThemeService` that facilitates dynamic theme switching across your Uno Platform applications. This service is especially useful for applications requiring runtime theme changes and for persisting user theme preferences.
+
+## Registering ThemeService
+
+To utilize the `ThemeService` in your application, you must register it during the app startup phase. Here’s how to set up the `ThemeService`:
+
+### App Startup Configuration
+
+1. **Register ThemeService**:
+   
+   Add the `ThemeService` to your project's host builder configuration.
+
+   ```csharp
+   public partial class App : Application
+   {
+       protected override void OnLaunched(LaunchActivatedEventArgs args)
+       {
+           var builder = this.CreateBuilder(args)
+               .Configure(host => host
+                   .UseThemeSwitching()
+               );
+       }
+   }
+    ```
+2. **Consume ThemeService**:
+
+    Inject the ThemeService into your view models or other services where you need to manipulate the theme.
+
+    ```csharp
+
+    public class SettingsViewModel
+    {
+        private readonly IThemeService _themeService;
+
+    public SettingsViewModel(IThemeService themeService)
+        {
+            _themeService = themeService;
+        }
+
+        public async Task ToggleThemeAsync()
+        {
+            var currentTheme = _themeService.CurrentTheme;
```

Comment:
I think it's _themeService.Theme and not CurrentTheme

https://github.com/unoplatform/uno.extensions/blob/74a945f7b6bbd3bf62c9deb64403d1e2ea20caf7/src/Uno.Extensions.Core/Toolkit/IThemeService.cs#L16

Reply:
@agneszitte, @kazo0 can you confirm?

Reply:
Yes indeed, `Theme`

Reply:
Done, Thanks!